config/test_query.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Replace the keys with the simple English query. i.e. first entry will be "List number of records in table"